common/azure-common/pom.xml

```diff
@@ -106,57 +110,30 @@
     <dependency>
       <groupId>com.github.tomakehurst</groupId>
       <artifactId>wiremock-standalone</artifactId>
-      <version>2.15.0</version>
       <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.eclipse.jetty</groupId>
       <artifactId>jetty-http</artifactId>
-      <version>9.4.8.v20171121</version>
       <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.eclipse.jetty</groupId>
       <artifactId>jetty-server</artifactId>
-      <version>9.4.8.v20171121</version>
       <scope>test</scope>
     </dependency>
   </dependencies>
   <build>
     <plugins>
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-compiler-plugin</artifactId>
-        <version>3.1</version>
-        <configuration>
-          <source>1.8</source>
-          <target>1.8</target>
-        </configuration>
-      </plugin>
-
       <plugin>
         <groupId>org.codehaus.mojo</groupId>
         <artifactId>build-helper-maven-plugin</artifactId>
       </plugin>
 
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-javadoc-plugin</artifactId>
-        <version>2.8</version>
-        <configuration>
-          <excludePackageNames>*.implementation.*;*.utils.*;com.microsoft.schemas._2003._10.serialization;*.blob.core.storage</excludePackageNames>
-          <bottom><![CDATA[<code>/**
-<br />* Copyright (c) Microsoft Corporation. All rights reserved.
-<br />* Licensed under the MIT License. See License.txt in the project root for
-<br />* license information.
-<br />*/</code>]]></bottom>
-        </configuration>
-      </plugin>
-
+      <!--The plugin may outdated with WireWork. TODO: Move to WireWork -->
```

Comment:
I'm not sure what this comment is stating. Can you please clarify, and if necessary file a github issue and reference it from within the comment? Same with the equivalent comment below.

Reply:
Talked to @jianghaolu about the plugin. He suggested me to add the comments. I can add an issue for this.

Reply:
Create an [issue ](https://github.com/Azure/azure-sdk-for-java/issues/3255)and put it in comments. Please suggest if unnecessary, I will close the issue.